src/NzbDrone.Core/Notifications/Webhook/WebhookPayload.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace NzbDrone.Core.Notifications.Webhook
+{
+    internal class WebhookPayload
+    {
+        public string EventType { get; internal set; }
+        public WebhookSeries Series { get; internal set; }
+        public WebhookEpisodeFile EpisodeFile { get; internal set; }
+        public WebhookEpisode Episode { get; internal set; }
```

Comment:
if I remember correctly, as far as sonarr is concerned scene name is the name of the torrent/nzb, we don't actually use the internal filename of episodes inside of a pack, so `SceneName` should be unique. as for quality, that is a good point,

we could always move the quality from `ReleaseInfo` to `Episode`